scanpy/tests/test_pca.py

```diff
@@ -343,6 +344,12 @@ def test_mask(array_type):
     adata = sc.datasets.blobs(n_variables=10, n_centers=3, n_observations=100)
     adata.X = array_type(adata.X)
 
+    if isinstance(adata.X, np.ndarray) and Version(ad.__version__) < Version("0.9"):
+        pytest.xfail(
+            "TODO: Previous version of anndata would return an F ordered array for one"
+            " case here, which suprisingly considerably changes the results of PCA. "
+        )
```

Comment:
We use xfail-strict, so please do this instead:

```py
request.node.add_marker(pytest.mark.xfail(...))
```

that way, the test suite fails with XPASS if the test unexpectedly starts passing